common/src/main/scala/uk/ac/wellcome/platform/archive/common/ingests/services/IngestUpdater.scala

```diff
@@ -62,4 +67,29 @@ class IngestUpdater(
       subject = s"Sent by ${this.getClass.getSimpleName}"
     )
   }
+
+  val descriptionMaxLength = 250
+  private def eventDescription(main: String,
+                               maybeInformation: Option[String]): String = {
+    val separator: String = " - "
+    truncate(
+      Seq(
+        Some(main),
+        maybeInformation
+      ).flatten.mkString(separator),
+      descriptionMaxLength)
+  }
+
+  private def truncate(text: String, maxLength: Int): String = {
```

Comment:
This is good, but I’d throw in a warning here if we're dropping descriptions that are >250 characters long, because we really shouldn't.

(This means not including S3 keys in messages, for example, although I think we're already doing that.)